src/app/pdf-downloader/page.tsx

```diff
@@ -1,23 +1,27 @@
 import { Metadata } from "next";
 import React, { ReactElement } from "react";
 import Title from "@/components/Title/Title";
-import Centerer from "@/app/pdfDownloader/pageUtils";
-import ShippingsLabel from "@/components/ShippingLabels/ShippingLabel";
+import Centerer from "@/app/pdf-downloader/pageUtils";
+import ShippingsLabel from "@/pdf/ShippingLabels/ShippingLabel";
+import ShoppingList from "@/pdf/ShoppingList/ShoppingList";
 
 const PdfDownloader = async (): Promise<ReactElement> => {
+    const TEST_PARCEL_ID = "85b7626f-a843-4d5e-9043-44a37a73c8aa";
     return (
         <main>
             <Title>PDF Downloader</Title>
             <Centerer>
-                {/* @ts-ignore */}
                 <ShippingsLabel text="Download Shipping Labels" />
             </Centerer>
+            <Centerer>
+                <ShoppingList text="Download Shopping Lists" parcelID={TEST_PARCEL_ID} />
+            </Centerer>
         </main>
     );
 };
 
 export const metadata: Metadata = {
-    title: "PdfDownloader",
+    title: "PDF Downloader",
```

Comment:
And also this - is the change necessary (?)

Reply:
I had a comment from Tom before that the metadata title should be "displayable"? (the `AddClients` was changed to `Add Clients` if i rmb correctly)

Reply:
yep - the metadata dictates what is in the 'title' - ie the text in the actual tab.
![image](https://github.com/VauxhallFoodbank/vauxhall-foodbank/assets/104017547/1bb7b5f2-4edb-440a-83c8-a721312a038f)
this change is good!